### PR TITLE
[13.0][IMP] stock_picking_batch_extended: Add confirm to validate and done buttons.

### PR DESCRIPTION
**Pull request by victoralmau**
Add confirm to validate and done buttons.

Please @pedrobaeza and @carlosdauden can you review it?

@Tecnativa TT30796

### CODE REVIEW

File: stock_picking_batch_extended/views/stock_batch_picking.xml [hunk 1]
```diff
@@ -22,13 +22,15 @@
                         string="Validate"
                         type="object"
                         attrs="{'invisible': ['|',('use_oca_batch_validation', '!=', True),('state', '!=', 'draft')]}"
+                        confirm="All pickings from this batch will be validate, are you sure?"
```
Comment (pedrobaeza) [inline stock_picking_batch_extended/views/stock_batch_picking.xml]:
```suggestion
                        confirm="All the pickings from this batch will be transferred. Are you sure?"
```
Don't forget the final `d` on passive participle and the article when specific.

File: stock_picking_batch_extended/views/stock_batch_picking.xml [hunk 2]
```diff
@@ -22,13 +22,15 @@
...
                     />
                     <button
                         name="action_transfer"
                         string="Validate"
                         class="oe_highlight"
                         type="object"
                         attrs="{'invisible': ['|',('use_oca_batch_validation', '!=', True), ('state', 'not in', ['in_progress','assigned'])]}"
+                        confirm="All pickings from this batch will be validate, are you sure?"
```
Comment (pedrobaeza) [inline stock_picking_batch_extended/views/stock_batch_picking.xml]: Same as previous.

File: stock_picking_batch_extended/views/stock_batch_picking.xml [hunk 3]
```diff
@@ -72,6 +74,7 @@
                         type="object"
                         class="oe_highlight"
                         attrs="{'invisible': ['|',('use_oca_batch_validation', '=', True), ('state', 'not in', ['in_progress','assigned'])]}"
+                        confirm="All pickings from this batch will be done, are you sure?"
```
Comment (pedrobaeza) [inline stock_picking_batch_extended/views/stock_batch_picking.xml]: You are changing the meaning of the button.

```suggestion
                        confirm="All the pickings from this batch will be confirmed. Are you sure?"
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**